contracts/BatchExchangeViewer.sol

```diff
@@ -1,6 +1,7 @@
 pragma solidity ^0.5.10;
 
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
```

Comment:
Perhaps we should include a note somewhere that OZ ^3.0 no longer has `ERC20Detaild` contract anymore.

Reply:
I think this will become obvious when we upgrade and get compilation errors. Do you have a reason why a reader of the current contracts should care?

Reply:
No I suppose not, I was thinking more of a TODO - placeholder. But its fine without.